src/lesson1/Loader.java

```diff
@@ -0,0 +1,11 @@
+package lesson1;
+
+public class Loader
+{
+    public static void main(String[] args)
+    {
+        Cat cat = new Cat();
+        System.out.println("Proverka Zheki");
+        System.out.println(cat.getStatus());
+    }
```

Comment:
Create 4 cats, show console result at comment in java code, like this // 4 cats created, 1 deleted

Reply:
done